src/Plugins/DBFTPlugin/Consensus/ConsensusService.Check.cs

```diff
@@ -45,8 +45,8 @@ private bool CheckPrepareResponse()
                 }
 
                 // Timeout extension due to prepare response sent
-                // around 2*15/M=30.0/5 ~ 40% block time (for M=5)
-                ExtendTimerByFactor(2);
+                // around 4*3/M=12.0/5 ~ 80% block time (for M=5)
+                ExtendTimerByFactor(4);
```

Comment:
Need dynamic calculation based on Native.PolicyContract.GetBlockGenTime().

Reply:
This is not good. That means can have 4*15s delay for 1 round if not enough PrepareResponse before SetBlockGenTime to 3s. I think keep it as 2 is not bad. We should only change variable but keep coefficient。